pandas/core/window.py

```diff
@@ -639,7 +684,7 @@ def aggregate(self, arg, *args, **kwargs):
     agg = aggregate
 
     @Substitution(name='window')
-    @Appender(_doc_template)
+#    @Appender(_doc_template)
```

Comment:
don't comment, simply remove

Reply:
Your comments are valuable and I have deleted necessary code.